src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -1887,6 +1887,9 @@ _reset (mongoc_client_pool_t **pool,
    *opts = mongoc_auto_encryption_opts_new ();
    extra = BCON_NEW ("mongocryptdBypassSpawn", BCON_BOOL (true));
    mongoc_auto_encryption_opts_set_extra (*opts, extra);
+#ifdef TESTING_CSFLE_OVERRIDE_PATH
```

Comment:
Good question. Because I want the acquisition of `csfle` to be fully automatable and human-error-free, and `csfle` is managed by another team entirely (i.e. driver devs won't likely be trying to iterate on it directly), it is built directly into the build process rather than needing to be specified manually. Of course, specifying an environment variable could also be done via scripting (e.g. with the `ENVIRONMENT` CTest property). This was the easiest way to get it working for now. In the future, one will likely want to support injecting alternative `csfle` libraries. I put this as YAGNI for now.